ansible/playbook.yml

```diff
@@ -270,6 +271,40 @@
       - pixelbook-touchscreen-click
       - pixelbook-keyboard-backlight
 
+    - name: Ensure keyboard hwdb is not overridden
+      become: true
+      file:
+        path: /etc/udev/hwdb.d/61-eve-keyboard.hwdb
+        state: absent
+      when: leftmeta == "capslock"
+
+    - name: Create keyboard hwdb override
```

Comment:
L281 -> L294 only runs if user sets the `leftmeta` var to `"leftmeta"` . If the user wanted to bind something _else_ to the `leftmeta` key, these steps would not run.

I think using `when: leftmeta != "capslock"` (or similar) might be better here.

Reply:
Sure. Seems reasonable!